src/services/admin/mentor.service.ts

```diff
@@ -75,3 +75,38 @@ export const getAllMentors = async (
     throw new Error('Error getting mentors')
   }
 }
+
+export const findAllMentorEmails = async (
+  status: ApplicationStatus | undefined
+): Promise<{
+  statusCode: number
+  emails?: string[]
+  message: string
+}> => {
+  try {
+    const mentorRepository = dataSource.getRepository(Mentor)
+
+    const allMentors: Mentor[] = await mentorRepository.find({
+      where: status ? { state: status } : {},
+      select: ['profile'],
+      relations: ['profile', 'category']
```

Comment:
We don't need the category relation here.

```suggestion
      relations: ['profile']
```